editor/src/components/menubar/menubar.tsx

```diff
@@ -368,6 +370,9 @@ export const Menubar = React.memo(() => {
           <Tile style={{ marginTop: 12, marginBottom: 12 }} size='large'>
             <a onClick={onTriggerSelectionTest}>P E</a>
           </Tile>
+          <Tile style={{ marginTop: 12, marginBottom: 12 }} size='large'>
+            <a onClick={onTriggerAbsoluteMoveTest}>PAM</a>
```

Comment:
![pan_or_pam](https://user-images.githubusercontent.com/1044774/157886502-3e3ccfda-ff39-49cf-b6b5-96b3648dee4e.gif)

Reply:
It's PAM.